From 201797df0d8b03375a30efc961d79148497d9de4 Mon Sep 17 00:00:00 2001
From: wlq <wlq@rock-chips.com>
Date: Thu, 24 Oct 2013 15:17:34 +0800
Subject: [PATCH] AndroidComputer: remote Kconfig default n modified: 
 arch/arm/configs/rk3188_ac_defconfig 	drivers/input/remotectl/Kconfig

---
 arch/arm/configs/rk3188_ac_defconfig | 2 ++
 drivers/input/remotectl/Kconfig      | 4 ++--
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/configs/rk3188_ac_defconfig b/arch/arm/configs/rk3188_ac_defconfig
index a0d9a46d887b..6808ca31ae93 100755
--- a/arch/arm/configs/rk3188_ac_defconfig
+++ b/arch/arm/configs/rk3188_ac_defconfig
@@ -254,6 +254,8 @@ CONFIG_GSENSOR_DEVICE=y
 CONFIG_COMPASS_DEVICE=y
 CONFIG_COMPASS_AK8963=y
 CONFIG_GYROSCOPE_DEVICE=y
+CONFIG_ROCKCHIP_REMOTECTL=y
+CONFIG_RK_REMOTECTL=y
 CONFIG_RK_IR_WAKEUP=y
 # CONFIG_SERIO is not set
 # CONFIG_VT is not set
diff --git a/drivers/input/remotectl/Kconfig b/drivers/input/remotectl/Kconfig
index b52921fc4b33..8f353cfc9453 100755
--- a/drivers/input/remotectl/Kconfig
+++ b/drivers/input/remotectl/Kconfig
@@ -3,7 +3,7 @@
 #
 menuconfig ROCKCHIP_REMOTECTL
 	bool "rkxx remotectl"
-	default y
+	default n
 	help
 	  Say Y here, will suport rk remotectl.
 	  This option doesn't affect the kernel.
@@ -13,7 +13,7 @@ if ROCKCHIP_REMOTECTL
 
 config RK_REMOTECTL
 	bool "rkxx remoctrl"
-	default y
+	default n
 	
 config RK_IR_WAKEUP
 	bool "rkxx remoctrl wakeup"
-- 
2.35.3

